app/code/core/Mage/Adminhtml/Block/Checkout/Formkey.php

```diff
@@ -30,7 +30,7 @@ class Mage_Adminhtml_Block_Checkout_Formkey extends Mage_Adminhtml_Block_Templat
      */
     public function canShow()
     {
-        return !Mage::getStoreConfigFlag('admin/security/validate_formkey_checkout');
+        return !Mage_Core_Controller_Front_Action::_isFormKeyEnabled();
```

Comment:
I would leave `isFormKeyEnabled` as-is and replace the config key with the constant `Mage_Core_Controller_Front_Action::XML_CSRF_USE_FLAG_CONFIG_PATH`.

Reply:
maybe we could have a middleground moving that logic to a helper method instead of Mage_Core_Controller_Front_Action?

Reply:
I believe we should have a public method to encourage (make life easier) for people implementing new forms to use the CSRF

Reply:
Sure, a core helper would be better.

Reply:
I've redone this part, what do you think about it?